helm/certs/gen_certs.sh

```diff
@@ -47,5 +38,6 @@ DNS.2 = $SERVICE_NAME-svc.$SERVICE_NAME
 DNS.3 = $SERVICE_NAME-svc.$SERVICE_NAME.svc
 DNS.4 = $SERVICE_NAME-svc.$SERVICE_NAME.svc.cluster.local
 EOF
-openssl req -new -key tls-key.pem -subj "/CN=$SERVICE_NAME-svc.$SERVICE_NAME.svc" -out tls.csr -config tls.conf
-openssl x509 -req -in tls.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out tls-crt.pem
+
+openssl req -new -key tls.key -sha256 -config tls-csr.conf -subj "/CN=$SERVICE_NAME-svc.$SERVICE_NAME.svc" -out tls.csr
+openssl x509 -req -days 365 -in tls.csr -signkey tls.key -sha256 -extensions v3_ca -extfile tls-crt.conf -out tls.crt
```

Comment:
Is 365 days enough / is there a rotation mechanism?

Reply:
Also, are we signing with the same key that the certificate is for? Please check with @Starkteetje if that's ok

Reply:
It's supposed to be a self-signed cert, right? In that case that's fine